Python/libraries/recognizers-date-time/recognizers_date_time/date_time/german/date_parser_config.py

```diff
@@ -193,9 +193,9 @@ def get_swift_day(self, source: str) -> int:
             swift = 0
         elif trimmed_text == 'morgen':
             swift = 1
-        elif trimmed_text == 'übermorgen':
+        elif trimmed_text == 'gestern':
```

Comment:
Ideally these should be changed to use YAML resources. Could you do it in a future PR?